src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -4110,18 +4110,19 @@ void SizeAirLoopBranches(EnergyPlusData &state, int const AirLoopNum, int const
         if (PrimaryAirSystems(AirLoopNum).DesignVolFlowRate == AutoSize) {
             CheckSysSizing(state, "AirLoopHVAC", PrimaryAirSystems(AirLoopNum).Name);
             PrimaryAirSystems(AirLoopNum).DesignVolFlowRate = FinalSysSizing(AirLoopNum).DesMainVolFlow;
-
-            {
-                auto const SELECT_CASE_var(FinalSysSizing(AirLoopNum).ScaleCoolSAFMethod);
-                if (SELECT_CASE_var == FlowPerFloorArea) {
-                    ScalableSM = "User-Specified(scaled by flow / area) ";
-                } else if (SELECT_CASE_var == FractionOfAutosizedCoolingAirflow) {
-                    ScalableSM = "User-Specified(scaled by fractional multiplier) ";
-                } else if (SELECT_CASE_var == FlowPerCoolingCapacity) {
-                    ScalableSM = "User-Specified(scaled by flow / capacity) ";
-                } else {
-                    ScalableSM = "Design ";
-                }
+            switch (FinalSysSizing(AirLoopNum).ScaleCoolSAFMethod) {
+            case FlowPerFloorArea: {
```

Comment:
Can probably be a `std::array<std::string_view>`. Does `default` ever actually happen?